docs/pages/contribute.mdx

```diff
@@ -32,7 +32,18 @@ The following steps are only necessary if you want to contribute to MUD. To use
    pnpm build
    ```
 
-### Pull requests
+### Using your repository in an application
+
+To check the changes in an application, run these commands at the application's root.
+
+```sh copy
+pnpm mud set-version --link <link to MUD repository>
+pnpm install
+```
+
+While the application's `pnpm dev` is running, any time you `pnpm build` the MUD repository the application gets updated automatically.
```

Comment:
it's also possible to run `pnpm dev` in the MUD repository to automatically rebuild on any changes in the MUD repository